src/main.tsx

```diff
@@ -14,11 +13,11 @@ import SharedProvider from './SharedProvider';
 
 const queryClient = new QueryClient();
 
-const localStoragePersister = createSyncStoragePersister({ storage: window.localStorage });
+/* const localStoragePersister = createSyncStoragePersister({ storage: window.localStorage });
 persistQueryClient({
   queryClient,
   persister: localStoragePersister,
-});
+}); */
```

Comment:
Why are we removing this?

Reply:
This happened in [this commit](https://github.com/pendulum-chain/portal/pull/326/commits/ad4ff0cec093abf0d33ff8c796ea33ab70131c19) and I don't know the reasoning for it either. Do we need this storage persister? Seems like it stores responses to fetch requests (indexer or RPC) in the local storage.